pgmpy/factors/CPD.py

```diff
@@ -369,6 +371,51 @@ def to_factor(self):
         return Factor(self.variables, self.cardinality, self.values)
 
 
+    def reorder_parents(self,new_order,inplace = False):
+        '''
+
```

Comment:
@anaviltripathi Please refer to docstrings of other methods and try to follow that exact same format. Since our online docs are automatically generated from the docstrings, changing the format here will mess up the docs. Also please add an examples section and add a few lines describing what this method does.